server/src/main/java/org/elasticsearch/indices/breaker/HierarchyCircuitBreakerService.java

```diff
@@ -251,6 +251,14 @@ public void checkParentLimit(long newBytesReserved, String label) throws Circuit
                 return;
             }
             this.parentTripCount.incrementAndGet();
+            LOGGER.warn(
+                "[Parent] New used memory {} [{}] for [{}] would be larger than configured breaker: {} [{}], breaking",
+                newBytesReserved,
+                ByteSizeValue.humanReadableBytes(newBytesReserved),
+                label,
+                parentLimit,
+                ByteSizeValue.humanReadableBytes(parentLimit)
+            );
```

Comment:
```suggestion
            if (LOGGER.isWarnEnabled()) {
                LOGGER.warn(
                    "[Parent] New used memory {} [{}] for [{}] would be larger than configured breaker: {} [{}], breaking",
                    newBytesReserved,
                    ByteSizeValue.humanReadableBytes(newBytesReserved),
                    label,
                    parentLimit,
                    ByteSizeValue.humanReadableBytes(parentLimit)
                );
            }
```

Otherwise `humanReadableBytes` is executed even if the log level is < warn